src/FormManager.php

```diff
@@ -432,6 +437,18 @@ private function prepare(): void
             }
         }
 
+        // If the last form field is not a page break, we need to merge the last step into the previous one
+        if (!$isPageBreakLastFormField) {
+            $lastStepIndex = count($this->formFieldsPerStep) - 1;
+
+            $this->formFieldsPerStep[$lastStepIndex - 1] = [
```

Comment:
What if those steps do not exist?

Reply:
Yeah this would happen only if there are no page breaks at all. I have just pushed another check.